ocpp-v1_6/src/test/java/eu/chargetime/ocpp/feature/profile/test/ServerFirmwareManagementProfileTest.java

```diff
@@ -48,12 +48,15 @@ public void setup() {
     }
 
     @Test
-    public void getFeatureList_containsGetDiagnosticsFeature() {
+    public void getFeatureList_containsAllNeededFeatures() {
         // When
         Feature[] features = profile.getFeatureList();
 
         // Then
-        assertThat(findFeature(features, "GetDiagnostics"), is(instanceOf(GetDiagnosticsFeature.class)));
+        assertThat(findFeature(features, "GetDiagnostics"), Is.is(instanceOf(GetDiagnosticsFeature.class)));
```

Comment:
Principle, only one assert per unit test. Why?
It makes the naming the unit test allot easier for starters... And since an assert is a throwable, you don't know it's only the first failing assert or all the asserts that fail. 
http://osherove.com/blog/2005/4/14/try-to-avoid-multiple-asserts-in-a-single-unit-test.html

Reply:
You're right, I'll fix that.